.github/workflows/nightly.yml

```diff
@@ -0,0 +1,35 @@
+name: Nightly Test
+
+on:
+  schedule:
+    - cron: '15 3 * * *'
+
+env:
+  COLORTERM: 'yes'
+  TERM: 'xterm-256color'
+  PYTEST_ADDOPTS: '--color=yes'
+  CONTAINER_RUNTIME: 'docker'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        image_tag:
+          - "latest"
```

Comment:
Do we want 3.9 here also, or are you assuming that "latest" is the same?

Reply:
It should be the same. In fact, there's no 3.9 container at the moment. I'm open to the idea of having one though.